### PR TITLE
[tone-analyzer] Support for v3

### PR DESCRIPTION
**Pull request by herchu**
Adds v3-experimental to Tone Analyzer (just released!)

### CODE REVIEW

**Comment by herchu** [comment]
Let us hold on this merge. For now we are not releasing V3 in the SDKs. 

The code will be useful anyway as soon as we do it (in January) so I suggest to leave the PR open.

File: README.md [hunk 1]
```diff
@@ -517,6 +518,25 @@ var params = {
...
+Use the [Tone Analyzer][tone_analyzer] service to analyze the
+emotion, writing and social tones of a text.
+
+```js
+var watson = require('watson-developer-cloud');
+
+var tone_analyzer = watson.tone_analyzer({
+  username: '<username>',
+  password: '<password>',
+  version: 'v2-experimental'
```
Comment (germanattanasio) [inline README.md]: I think this should be **v3** not **v2**

File: services/tone_analyzer/v3-experimental.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    return;
+  }
+
+  var parameters = {
+    options: {
+      url: '/v3/tone',
+      method: 'POST',
+      body: params.text
+    },
+    defaultOptions: extend(this._options, {
```
Comment (germanattanasio) [inline services/tone_analyzer/v3-experimental.js]: this should be:
```js
var parameters = {
      options: {
      url: '/v3/tone',
      method: 'POST',
      body: params.text,
      headers: {
         'accept':'application/json',
         'content-type': 'text/plain'
      },
      defaultOptions: this._options
}
```

File: README.md (hunk 1 above)
Comment (herchu) [inline README.md]: Ok - fixed

File: services/tone_analyzer/v3-experimental.js (hunk 2 above)
Comment (herchu) [inline services/tone_analyzer/v3-experimental.js]: Are you sure? See test cases... the version in my fork generates the right headers.... (perhaps testcases are wrong though)

**Comment by herchu** [comment]
Hey, I updated the codebase. The new module is now v3-beta. Can you please review and merge?

File: README.md (hunk 1 above)
Comment (germanattanasio) [inline README.md]: The example in the README needs to be updated

See: https://github.com/herchu/nodejs-wrapper/tree/dev#tone-analyzer

Comment (herchu) [inline README.md]: Bleh, I had several committed changes I had not pushed! That one is fixed now

**Comment by germanattanasio** [comment]
@herchu I just realized that you are not checking for the `version_date`

**Pull request merged**